frontend/app/forms/MemberForm.scala

```diff
@@ -119,7 +138,8 @@ object MemberForm {
       "marketingChoices" -> marketingChoicesMapping,
       "password" -> optional(nonEmptyText),
       "casId" -> optional(nonEmptyText),
-      "subscriberOffer" -> default(boolean, false)
+      "subscriberOffer" -> default(boolean, false),
+      "featureChoice" -> productFeature
```

Comment:
An interesting thing that's going on here is that I think that there's no server-side validation that a legitimate productFeature selection has been made. Specifically, a user signing up as a 'Partner' could manipulate the form so that it posted _both_ product features (Books *and* Free event tickets) - and they would get them, despite not signing up as a Patron.